Exiled.Events/Handlers/EventArgs/LocalReportEventArgs.cs

```diff
@@ -0,0 +1,49 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="LocalReportEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Handlers.EventArgs
+{
+    using System;
+
+    /// <summary>
+    ///     Contains information about the report to local administrators.
+    /// </summary>
+    public class LocalReportEventArgs : EventArgs
+    {
+        /// <param name="issuer"><inheritdoc cref="Issuer"/></param>
+        /// <param name="target"><inheritdoc cref="Target"/></param>
+        /// <param name="reason"><inheritdoc cref="Reason"/></param>
+        /// <param name="isAllowed"><inheritdoc cref="IsAllowed"/></param>
+        public LocalReportEventArgs(API.Features.Player issuer, API.Features.Player target, string reason, bool isAllowed = true)
+        {
+            Issuer = issuer;
+            Target = target;
+            Reason = reason;
+            IsAllowed = isAllowed;
+        }
+
+        /// <summary>
+        ///     Report sender.
+        /// </summary>
+        public API.Features.Player Issuer { get; internal set; }
```

Comment:
You can directly put `using Exiled.API.Features` above, no to rewrite the namespace every time.